api/jwt/jwt.go

```diff
@@ -164,6 +166,12 @@ func (service *Service) generateSignedToken(data *portainer.TokenData, expiresAt
 		return "", fmt.Errorf("invalid scope: %v", scope)
 	}
 
+	if _, ok := os.LookupEnv("DOCKER_EXTENSION"); ok {
+		// Set expiration to 99 years for docker desktop extension.
+		log.Infof("[message: using 99 year JWT expiration time for docker desktop extension environment]")
```

Comment:
I don't think we should reveal this.   Rather just reveal that we're running in docker desktop extension mode